lib/assessment.rb

```diff
@@ -43,6 +45,6 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+   File.open(filename).map(&:chomp).join(" ")
```

Comment:
Hah! Clever - since the newline character will always be last in the line, `chomp` will get rid of every one, and using will ensure that there's no leftover whitespace at the end (unless, of course, there's trailing whitespace in the last line.... but let's ignore that case for now).